.changes/fix-notarytool-args-apple-id.md

```diff
@@ -0,0 +1,5 @@
+---
+'tauri-cli': 'patch:bug'
```

Comment:
i think this also needs to mention `@tauri-apps/cli` if we wanna release this as a quick hotfix 🤔 Still super unsure about our changefile system after all this time lol